webpack/assets/javascripts/react_app/components/common/forms/FormField.js

```diff
@@ -0,0 +1,162 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+  Col,
+  FormGroup,
+  FormControl,
+  ControlLabel,
+  HelpBlock,
+  FieldLevelHelp,
+} from 'patternfly-react';
+
+import { noop } from '../../../common/helpers';
+import DateTimePicker from '../DateTimePicker/DateTimePicker';
+import DatePicker from '../DateTimePicker/DatePicker';
+import TimePicker from '../DateTimePicker/TimePicker';
+
+const inputComponents = {
+  date: DatePicker,
+  dateTime: DateTimePicker,
+  time: TimePicker,
+};
```

Comment:
Does it make sense to add helpers for the rest of input components?

Reply:
1. I wouldn't do it now,
2. It's upon discussion if to support all the inputs from rails, or just those which are not accessible now (rich components) and as you've stated already above from React we can just easily use the `children` prop for other components.

Will do a followup and discuss there, if you agree :slightly_smiling_face: